Handler/Cancel.hs

```diff
@@ -0,0 +1,22 @@
+module Handler.Cancel where
+
+import Import
+import Stripe
+
+postCancelR :: Handler Html
+postCancelR = do
+    plan <- runDB $ getBy404 $ UniquePlan freePlanId
```

Comment:
It does. The normal `get` and `getBy` functiosn return `Maybe`, while `get404` and `getBy404` don't, but short-circuit the handler with a 404 status when the record's not found.

https://www.fpcomplete.com/hoogle?q=getBy404&env=ghc-7.8-stable-14.09